waiter/integration/waiter/token_request_test.clj

```diff
@@ -595,8 +693,9 @@
         (testing "token retrieval"
           (let [token-response (get-token waiter-url token)
                 response-body (-> token-response (:body) (json/read-str) (pc/keywordize-map))]
-            (is (= (assoc service-description :authentication "disabled" :owner current-user)
-                   response-body))))
+            (is (contains? response-body :last-update-time))
+            (is (= (assoc service-description :authentication "disabled" :deleted false :owner current-user)
```

Comment:
Why is `deleted` in the response?  Are normal users going to start seeing `deleted`?  The only time it should show up is when `show-deleted` is `true` and the token is deleted.